MultiVersions/Forge/main/beachparty/lang/zh_tw.json

```
@@ -49,95 +49,77 @@
   "block.beachparty.radio": "收音機",
   "block.beachparty.sand_bucket": "沙桶",
   "block.beachparty.sand_bucket_block": "沙桶",
-  "block.beachparty.sand_dirty": "帶雜物的沙",
   "block.beachparty.sand_pile": "沙堆",
-  "block.beachparty.sand_seastars": "帶海星的沙",
-  "block.beachparty.sand_slab": "沙半磚",
   "block.beachparty.sandcastle": "沙堆城堡",
-  "block.beachparty.sandwaves": "波浪沙",
-  "block.beachparty.stripped_palm_log":"剝皮棕櫚木原木",
-  "block.beachparty.stripped_palm_wood":"剝皮棕櫚木塊",
-  "block.beachparty.sweetberries_cocktail": "草莓雞尾酒",
+  "block.beachparty.sandwaves": "海浪沙",
+  "block.beachparty.stripped_palm_log": "剝皮棕櫚木原木",
+  "block.beachparty.stripped_palm_wood": "剝皮棕櫚木塊",
+  "block.beachparty.sweetberries_cocktail": "甜莓雞尾酒",
   "block.beachparty.table": "棕櫚木桌",
-  "block.beachparty.tiki_bar":"熱帶風格吧檯",
-  "block.beachparty.tiki_chair":"熱帶風格木椅",
-  "entity.beachparty.pelican": "鵜鶘",
+  "block.beachparty.tiki_bar": "熱帶風格吧檯",
+  "block.beachparty.tiki_chair": "熱帶風格吧檯椅",
+  "config.beachparty.subCategory.beach": "沙灘派對套裝",
+  "creativetab.beachparty.tab": "共度時光：沙灘派對",
```

itemGroup 我們習慣用 `｜` 分隔
```suggestion
  "creativetab.beachparty.tab": "共度時光｜沙灘派對",
```